docs/examples/example.ipynb

```diff
@@ -335,32 +333,87 @@
     "```python\n",
     "# wifi-sensor/src/wifi_sensor.py\n",
     "from viam.resource.registry import Registry, ResourceCreatorRegistration\n",
-    "from viam.components.sensor import Sensor\n",
     "\n",
     "async def main():\n",
     "    Registry.register_resource_creator(Sensor.SUBTYPE, MySensor.MODEL, ResourceCreatorRegistration(MySensor.new))\n",
-    "```\n"
+    "```\n",
+    "\n",
+    "Click [here](https://gist.github.com/purplenicole730/dc3ddd9d53fd5da16c3bde0e0a6a63f4#file-module_step2-py) to see the file with its new `MODEL` variable, creator function, and `main()` function.\n",
```

Comment:
This is an intermediary step, so we're showing what it looks like right after registering the resource in the registry. @dannenberg requested to have snapshots in the middle of the steps so that a user can easily follow along.

Reply:
@njooma https://github.com/viamrobotics/viam-python-sdk/blob/fc86a2b25d0fcb5edc834e5c88bc4b3f838df822/docs/examples/example.ipynb#L333 for more context